src/ui/components/filters/sortoptionscomponent.js

```diff
@@ -21,9 +22,16 @@ export default class SortOptionsComponent extends Component {
     this.options[this.selectedOptionIndex].isSelected = true;
     this.hideExcessOptions = this._config.showMore && this.selectedOptionIndex < this._config.showMoreLimit;
     this.showReset = this._config.showReset && this.selectedOptionIndex !== 0;
+
+    /**
+     * This component listens to updates to vertical results, and sets its state to it when
+     * an update occurs.
+     * @type {string}
+     */
+    this.moduleId = StorageKeys.VERTICAL_RESULTS;
```

Comment:
This makes me a bit nervous. Are we certain this won't have unintended consequences? This component will get re-rendered every time there is an update to the VerticalResults.

Reply:
Double checked this by 
1. click show more
2. choose non default sort option within show more
3. ran normal search, sort option did not change
4. ran no results search, sort option disappears
5. ran another normal search, sort option came back with same show/more and selected option state

I think this makes sense since everything in the sort options component is tied pretty closely to a variable stored on the component like this.selectedOptionIndex. I should have checked this before though.;